packages/components/src/utils/style-mixins.js

```diff
@@ -1,6 +1,5 @@
 export { color, rgba } from './colors';
 export { reduceMotion } from './reduce-motion';
-export { rtl, getRTL, useRTL } from './rtl';
```

Comment:
As far as I can tell `rtl` is used in some places when handling CSS in JS. I reason based on failing unit tests.